README.md

```diff
@@ -5,9 +5,9 @@
 ## Requirements
 
 - go v1.12+
-- dep v0.5.x
-- operator-sdk v0.9.0
-- ocp 4.x
+- [dep](https://github.com/golang/dep#installation) v0.5.x
+- [operator-sdk](https://github.com/operator-framework/operator-sdk/releases) v0.9.0
+- ocp 4.x (you can use [CRC](https://github.com/code-ready/crc) for local deployment)
 - [kogito s2i imagestreams](https://raw.githubusercontent.com/kiegroup/kogito-cloud/master/s2i/kogito-imagestream.yaml) installed
 
```

Comment:
WDYT about add here first the following topics?
* Overview ( what and propose of this project )
* Getting started ( steps to getting started with )

Reply:
Acked! The make functions we'll track in a new JIRA!